src/ui/server/controllers/insurance/your-buyer/credit-insurance-cover/validation/rules/credit-insurance-cover-description.ts

```diff
@@ -26,11 +23,7 @@ const MAXIMUM = 1000;
  */
 const creditInsuranceCoverDescriptionRule = (formBody: RequestBody, errors: object) => {
   if (formBody[HAS_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER] === 'true') {
-    if (objectHasProperty(formBody, FIELD_ID)) {
-      return maxLengthValidation(formBody[FIELD_ID], FIELD_ID, ERROR_MESSAGE.ABOVE_MAXIMUM, errors, MAXIMUM);
-    }
-
-    return generateValidationErrors(FIELD_ID, ERROR_MESSAGE.IS_EMPTY, errors);
+    return providedAndMaxLength(formBody, FIELD_ID, ERROR_MESSAGES_OBJECT, errors, MAXIMUM_CHARACTERS.BUYER_PREVIOUS_CREDIT_INSURANCE_COVER_WITH_BUYER);
```

Comment:
dereference?

Reply:
Same as above - update this to be `BUYER.PREVIOUS_CREDIT_INSURANCE_COVER` though.